src/js/controllers/modals/search.js

```diff
@@ -77,4 +90,14 @@ angular.module('copayApp.controllers').controller('searchController', function($
     $scope.txHistoryShowMore = $scope.filteredTxHistory.length > $scope.txHistorySearchResults.length;
   };
 
+  $scope.searchOnBlockchain = function(searchTerm) {
+    var url = 'https://explorer.bitcoin.com/'+$scope.wallet.coin+'/search/' + searchTerm;
```

Comment:
Aren't we using the google style guide which says always let or const, never var?

Reply:
I think `uglify-js` is being used for production builds and it does not support ES6 syntax, it throws errors during the build process. Perhaps we should do the refactoring to google style guide in a separate task/branch, but it might also introduce conflicts with other branches as it will be a codebase-wide change..